src/libraries/System.Security.Cryptography.Cng/tests/CngKeyTests.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Xunit;
+using Xunit.Sdk;
+
+namespace System.Security.Cryptography.Tests
+{
+    public static class CngTests
```

Comment:
Nit: Class name should probably be `CngKeyTests`.

Reply:
Copy and paste error on my part. Doh. :(